src/components/Modal/ModalContainer.tsx

```diff
@@ -32,17 +33,27 @@ const ModalContainer: React.FunctionComponent<Props> = ({
 }: Props) => {
   return (
     <ModalOverlay {...props} onMouseDown={toggleModal}>
-      <ModalWrapper onClick={handleClick} onMouseDown={handleClick} size={size}>
-        <Content
-          toggleModal={toggleModal}
-          closeModal={closeModal}
-          keepOpenLabel={keepOpenLabel}
-          keepOpenCheckbox={
-            <Input inlineLabel name="createAnother" label={keepOpenLabel} type={'checkbox'} onChange={toggleKeepOpen} />
-          }
-          {...contentProps}
-        />
-      </ModalWrapper>
+      <div onClick={handleClick} onMouseDown={handleClick}>
+        <Draggable>
```

Comment:
Might be a good idea to use a handle instead of making the entire modal a handle.
I'm not sure, but as it is now, it might give issues for forms or other elements not responding to user input.

Reply:
done